SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -361,16 +361,23 @@ void USpatialNetDriver::OnConnectionToSpatialOSSucceeded()
 	{
 		Sender->CreateServerWorkerEntity();
 	}
+
+	if (USpatialGameInstance* GameInstance = GetGameInstance())
```

Comment:
I think this should be a `check` on the GameInstance instead of an if. If we don't have a GameInstance at this point we're in big trouble.

Reply:
Changed to `check` instead.